communitiesuk/funding-service-design-assessment
# Fs 2788 add dashboard stats for team flag

### Change description
Added frontend boxes to show stats for each Team if they have a flag assigned to them. For every team, a dashboard box is added to them

### How to test
Select an application, flag the application and assign it to a team, the dashboard should now add a box for this team showing how many flags are assigned to them.

### Screenshots of UI changes (if applicable)
![image](https://github.com/communitiesuk/funding-service-design-assessment/assets/60141291/a1660de3-5220-451c-9f03-93cb6728b207)
